### PR TITLE
Plans: Replace compare plans Noticon with Gridicon

### PR DESCRIPTION
**Pull request by breezyskies**
Before | After
------------ | -------------
screen shot 2015-12-04 at 11 24 32 am | screen shot 2015-12-04 at 11 22 04 am

Side note: This should probably be componentized. cc: @scruffian

### CODE REVIEW

**Comment by folletto** [comment]
Can we use size 18px instead of 16px (ideally 24px)? We're trying to standardize there :) 

Apart from that, looks good. :+1:

**Comment by breezyskies** [comment]
> Can we use size 18px instead of 16px (ideally 24px)?

Updated. :)

**Comment by alternatekev** [comment]
So much better! :+1:

**Comment by alternatekev** [comment]
The code for this looks good to me.

File: client/signup/steps/plans/index.jsx [hunk 1]
```diff
@@ -91,7 +92,8 @@ module.exports = React.createClass( {
 					href={ this.comparePlansUrl() }
 					className='plans-step__compare-plans-link'
 					onClick={ this.handleComparePlansLinkClick.bind( null, 'footer' ) }>
-					{ this.translate( 'Compare Plans' ) }
+						<Gridicon icon="clipboard" size="18" />
```
Comment (mtias) [inline client/signup/steps/plans/index.jsx]: For good measure, the size prop should be passed as a number not a string: `size={ 18 }`

**Pull request merged**